src/components/structures/LeftPanel.tsx

```diff
@@ -116,8 +118,9 @@ export default class LeftPanel extends React.Component<IProps, IState> {
         dis.fire(Action.OpenDialPad);
     };
 
-    private onExplore = () => {
+    private onExplore = (ev: ButtonEvent) => {
         dis.fire(Action.ViewRoomDirectory);
+        PosthogTrackers.trackInteraction("WebLeftPanelExploreRoomsItem", ev);
```

Comment:
Hmm `...Item` is normally used for context menu items, not for buttons which are visible at the top level

Reply:
That’s a good point, though we probably should’ve caught that mistake in https://github.com/matrix-org/matrix-analytics-events/pull/54 ?

Reply:
I think it isn't up to the team reviewing the Analytics events repo for checking where the app has a given button, especially when the partner PR doesn't exist to see where you'll hook it up.